src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -23,30 +23,37 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <!-- Shared -->
   <ItemGroup>
-    <Compile Include="..\..\Common\src\System\Numerics\Hashing\HashHelpers.cs">
+    <Compile Include="$(CommonPath)\System\Numerics\Hashing\HashHelpers.cs">
       <Link>Common\System\Numerics\Hashing\HashHelpers.cs</Link>
     </Compile>
-    <Compile Include="System\Numerics\ConstantHelper.cs">
+    <Compile Include="$(CommonPath)\CoreLib\System\Numerics\ConstantHelper.cs">
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
       <DependentUpon>ConstantHelper.tt</DependentUpon>
+      <Link>System\Numerics\ConstantHelper.cs</Link>
     </Compile>
-    <Compile Include="System\Numerics\JitIntrinsicAttribute.cs" />
-    <Compile Include="System\Numerics\Register.cs">
+    <Compile Include="$(CommonPath)\CoreLib\System\Numerics\JitIntrinsicAttribute.cs">
```

Comment:
We should not need the JitIntrinsicAttribute. As far as I know, this attribute was future proofing, never used for anything real. We should delete it, and use the IntrinsicAttribute from CoreLib instead.

Reply:
It looks like nothing is using it in `dotnet/coreclr`.  @CarolEidt - do you agree we can just delete the `JitIntrinsicAttribute`?  I was just moving code so it would compile, since `Vector<T>` has this attribute on its methods

Reply:
Yes, I would agree that we can delete the `JitIntrinsicAttribute`. When we move these to use the named intrinsic capabilities (which I assume we will do at some point "soon") then they should have the `[Intrinsic]` attribute. But the `JitIntrinsicAttribute` was basically just for documentation, IIRC.